tests/testFileEditor.js

```diff
@@ -212,6 +215,20 @@ const verifyEditData = [
 describe('test file editor', function() {
     this.timeout(20000);
 
+    describe('test base64-util lib, on which file editor depends', function() {
+        it('should encode a challenging string without crashing', function() {
```

Comment:
Nit: I think we should be more conservative with our usage `it(...)` blocks; this is the main reason our test output is so long. Not as important for just three tests, but in general I want to move towards grouping more testing inside a single `it(...)` block.

Reply:
fair enough, and this test has been removed anyway